cpp/src/strings/convert/convert_durations.cu

```diff
@@ -389,7 +350,29 @@ struct duration_to_string_fn : public duration_to_string_size_fn<T> {
     return ptr;
   }
 
-  __device__ void operator()(size_type idx)
+  __device__ size_type string_size(size_type idx)
+  {
+    if (d_durations.is_null(idx)) return 0;
```

Comment:
```suggestion
```
This is redundant since it is already handled by the caller. Since the `idx` parameter is only used to retrieve the `duration`, perhaps the calling function can resolve and just pass this instead.
This comment applies to other similar change patterns in this PR as well.

Reply:
Made the requested changes. Not sure I got the `idx` parameter suggestion right, though.